extensions/frontend-tiles/src/integration-test/utils/webpack-config.js

```diff
@@ -0,0 +1,126 @@
+// /*---------------------------------------------------------------------------------------------
```

Comment:
webpack-config is here.

Reply:
i think you want to use the [CopyExternalsPlugin](https://github.com/iTwin/itwinjs-core/blob/master/tools/webpack-core/src/plugins/CopyExternalsPlugin.ts#L94)

Reply:
I've added the plugin, I'm not sure if it's done correctly, I couldn't find examples of it being used in a webpack-config. If it is set up correctly, it doesn't seem to have done much. By removing the call to externalsPreset{node: true}, webpack still doesn't successfully compile/bundle, citing the same errors as before. The NodePolyfillPlugin above it does help to reduce the number of errors signigicantly (250 -> 110), but there are still many present. When the externalsPreset is present, webpack still compiles correctly, but then I cant run the tests and I get the same "require is not defined" error.